### PR TITLE
remove characters from value mapping

### PR DESCRIPTION
**Pull request by ebattat**
## Type of change
Note: Fill **x** in []
- [x] bug
- [ ] enhancement
- [ ] documentation
- [ ] dependencies

## Description

Removing characters from product versions

### CODE REVIEW

File: benchmark_runner/common/grafana/update_grafana_latest_value_mappings.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ class UpdateGrafanaLatestValueMappings:
     """
     # Prevent error messages from being saved instead of versions in case of a connection issue
     MAX_VERSION_LEN = 20
-    LAST_ES_FETCH_DAYS = 30
+    LAST_ES_FETCH_DAYS = 14
```
Comment (RobertKrawitz) [inline benchmark_runner/common/grafana/update_grafana_latest_value_mappings.py:15]: This change is unrelated and should be a separate PR if it's needed.

File: benchmark_runner/common/grafana/update_grafana_latest_value_mappings.py [hunk 2]
```diff
@@ -44,7 +44,8 @@ def get_last_elasticsearch_versions(self, last_es_fetch_days=LAST_ES_FETCH_DAYS)
             data = self.elasticsearch.get_elasticsearch_index_by_id(index='ci-status', id=id)
             for version, value in data['_source'].items():
                 if version in display_versions and value not in new_versions.values() and len(value) < self.MAX_VERSION_LEN:
-                    new_versions[value.replace(".", "").replace("-", "")] = value
+                    # Display only numbers in the Grafana panel, removing characters
+                    new_versions[value.replace(".", "").replace("-", "").replace("rc", "").replace("ec", "").replace("fc", "")] = value
```
Comment (RobertKrawitz) [inline benchmark_runner/common/grafana/update_grafana_latest_value_mappings.py:48]: Can you do this either via a regex or by a list of replacements?  This kind of code is hard to read.

**Review by RobertKrawitz — CHANGES_REQUESTED** [review]

File: benchmark_runner/common/grafana/update_grafana_latest_value_mappings.py (hunk 1 above)
Comment (ebattat) [reply benchmark_runner/common/grafana/update_grafana_latest_value_mappings.py]: Agree, remove

File: benchmark_runner/common/grafana/update_grafana_latest_value_mappings.py (hunk 2 above)
Comment (ebattat) [reply benchmark_runner/common/grafana/update_grafana_latest_value_mappings.py]: Fixed
